src/__test__/components/plots/FrequencyPlot.test.jsx

```diff
@@ -39,8 +41,11 @@ describe('Frequency plot tests', () => {
     fetchMock.mockIf(/.*/, mockAPI(generateDefaultMockAPIResponses(fake.EXPERIMENT_ID)));
   });
 
-  it('Updates CSV data on render', () => {
-    renderFrequencyPlot();
+  it('Updates CSV data on render', async () => {
+    await act(async () => {
+      renderFrequencyPlot();
+    });
```

Comment:
Do we need this? I think I can see that UI changes inside render is wrapped in `act()`: https://github.com/testing-library/react-testing-library/blob/main/src/pure.js#L109-L115

Reply:
Yes, it fails if not because it doesn't wait for the cellSets to load

Reply:
Oh ok. Can you move this `await act()` inside `renderFrequencyPlot`? That way this await doesn't have to be repeated for all calls to `renderFrequencyPlot`.